### ISSUE TITLE
ResourceMapFunction serializes timestamps as longs

### ISSUE THREAD

**Issue by lhazlewood** [stormpath/stormpath-sdk-java#1074]
For example, resulting JSON:

``` javascript
{
    "createdAt": 1475170548976,
    "modifiedAt": 1477515763694,
    // etc...
}
```

The values should be ISO 8601 strings, not longs. 

Upon further inspection, instead of using Java reflection to get a value, I think(?) we should be getting the underlying DefaultResource JSON map and using that instead.

**Comment by mrioan** [comment]
@lhazlewood is working on a broader solution for the serialization logic. I could not find the specific issue number for that task. This issue remains block until it gets done.

**Comment by lhazlewood** [comment]
This was resolved in 1.3.0.  ResourceMapFunction was deprecated in that release in favor of ResourceConverter.  ResourceConverter does indeed represent date instances as ISO-8601 formatted strings.
